components/raftstore/src/store/msg.rs

```diff
@@ -39,6 +41,13 @@ pub struct WriteResponse {
     pub response: RaftCmdResponse,
 }
 
+// Peer's internal stat, for test purpose only
+#[derive(Debug)]
```

Comment:
it cannot because it will be used by components/test_raftstore/src/cluster.rs which is not under #[cfg(test)].
Also the code will be a bit ugly because  #[cfg(test)] will scatter among product code.

Reply:
You can guard it with testexport features.

Reply:
> You can guard it with testexport features.

good to know. Thanks.